package Exception_handling;

public class test2 {

	public static void main(String[] args) {
	
		test1.main(null);

	}

}
